README.md

```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
 
 Connect to your ATD Lab environment.  If you need an ATD Lab instance, please contact your local account team.  Once connected to the ATD Lab instance, select the Programmability IDE.  This container is built with all the necessary requirements and python modules to run AVD playbooks.
 
+1. Setup Git user and email and setup the ATD lab environement.
 
-```shell
-# Setup your git global config (optional)
-git config --global user.email "you@example.com"
-git config --global user.name "Your Name"
+    - Open terminal window in vscode View -> Terminal from menu, and run the following commands:
 
-# Run Script to setup environment
-curl -fsSL https://get.avd.sh/atd/install.sh | sh
+    ```shell
+    # Setup your git global config (optional)
+    $ git config --global user.email "you@example.com"
+    git config --global user.name "Your Name"
 
-# Move to directory
-cd labfiles/arista-ansible/atd-avd
+    # Run Script to setup environment
+    $ curl -fsSL https://get.avd.sh/atd/install.sh | sh
+    ```
 
-# Update Inventory with Lab Credentials
-edit credentials in vscode: atd-avd/atd-inventory/inventory.yml
+2. Update Inventory with Lab Credentials
 
-# Run Playbook to Prepare CloudVision for AVD
-$ ansible-playbook playbooks/atd-prepare-lab.yml
+    - Open file in vscode: atd-avd/atd-inventory/inventory.yml
 
-# Execute Tasks in CVP manually
+    - Edit credentials with the ones provide on test drive landing page.
```

Comment:
- Set credentials in the `inventory.yml` to the credentials on the Arista Test Drive landing page.